BrickBrackBruck/BrickBrackBruck.csproj

```diff
@@ -13,5 +13,6 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="System.ValueTuple" Version="4.3.0"/>
+    <PackageReference Include="System.Collections.NonGeneric" Version="4.3.0"/>    
```

Comment:
Non-Gneric wird für ArrayList verwendet. Ist hier interessanterweise in einem anderen Paket
als beim MS Visual Studio 20XX. Fliegt raus, wenn wir die nicht benutzen.